types/src/jsonrpc/wrapped/batches.rs

```diff
@@ -86,7 +82,7 @@ pub enum BatchesEvent<'a, T> {
 /// Request within the batches.
 pub struct BatchesElem<'a, T> {
```

Comment:
Will rename to `BatchesRequest`. There is some more renaming to be done, since there is also a `batch` module that has a `BatchElem` type too, and I don't want to blow this PR too much.

Reply:
Actually scrap that, renamed the original `BatchElem` to `BatchRequest` too. The comment above here wasn't fitting after changes so reverted. Hope it all makes some sense now :).